docs/data/data-grid/localization/data.json

```diff
@@ -5,262 +5,262 @@
     "localeName": "Arabic (Sudan)",
     "missingKeysCount": 3,
     "totalKeysCount": 117,
-    "githubLink": "https://github.com/mui/mui-x/blob/master/packages/x-data-grid/src/locales/arSD.ts"
+    "githubLink": "https://github.com/mui/mui-x/blob/master/packages\\x-data-grid\\src\\locales\\arSD.ts"
```

Comment:
Please merge the latest `master` branch into yours and re-run `yarn l10n`.
These paths should no longer be generated incorrectly. 🤞